packages/idx/src/property/search.component.ts

```diff
@@ -282,6 +282,8 @@ Stratus.Components.IdxPropertySearch = {
                                         // console.log('updating json', scopeVarPath, 'value to', value, 'was', varElement.val())
                                         varElement.val(JSON.stringify(value))
                                     }
+                                    // varElement.fireEvent('onchange') // deprecated and no longer works
+                                    varElement[0].dispatchEvent(new Event('change'))
```

Comment:
This is a good equivalent, as it allows normal DOM binds to trigger appropriately when bubbling.